README_validations.md

```diff
@@ -0,0 +1,116 @@
+
+## OSCAL "complete" schema
+
+Additionally, there are "complete" XML and JSON schemas, which can validate any kind of OSCAL. It is larger and more complex than the model-specific schemas and may not be as efficient, but it can be convenient to use one schema file as an entrypoint to validate one or more OSCAL documents without knowing the particular model in advance, whether it be a `component-definition`, `system-security-plan`, `assessment-plan`, et cetera.
+
+## How do I apply a schema to a document to determine validity?
+
+Schemas are built using standard schema languages that are supported by commodity tools. These tools can all apply the same schemas to the same documents, and give the same results.
+
+For OSCAL XML, this schema syntax is [XML Schema Definition (XSD)](https://www.w3.org/TR/xmlschema-1/). For OSCAL JSON, the schema syntax is [JSON Schema](https://json-schema.org/). Many freely available, open-source, and commercial implementations are available. These schema standards are also built in to many common application frameworks and development environments.
+
+Each tool will have its own way to provide for validation. Bindings or configurations to link documents to schemas can be maintained either inside or outside the documents themselves; many tools offer several ways to do this.
+
+Some tools support self-contained validation functionality without full-fledged application frameworks or development environments. Validation against a XML schema in XSD syntax can be done in a single command, for example, with a tool like [`xmllint`](http://xmlsoft.org/xmllint.html) (where `oscal-schema.xsd` is the appropriate schema and `oscal.xml` is an example OSCAL document).
+
+```
+$ xmllint oscal.xml --schema oscal-schema.xsd --noout
+```
+
+Similarly, using the [`ajv-cli`](https://github.com/ajv-validator/ajv-cli), you can validate a JSON document like so:
+
+```
+# Install schema type library in NodeJS runtime to avoid runtime errors
+# and run the ajv command line tool after installation.
+npm install -g ajv-formats && npx ajv validate -s oscal-schema.json -d oscal.json -c ajv-formats
+```
+
+## What does success look like?
+
+If you validate an OSCAL catalog with its schema (a local copy) as above (using `xmllint`), it will return results to the standard output (STDOUT) stream like so:
+
+```
+$ xmllint oscal.xml --schema oscal-schema.xsd --noout
+oscal.xml validates
+```
+
+There is no detailed output, just a statement that indicates validation has been performed successfully.
+
+If you remove the `--noout` flag from the command line, a copy of the input will be returned. This is because the utility is designed for a pipeline architecture. Its outputs will be provided as inputs to the next tool in the pipeline.
+
+As for OSCAL JSON, the results from validation with AJV will look like this:
+
+```
+npm install -g ajv-formats && npx ajv validate -s oscal-schema.json -d oscal.json -c ajv-formats
```

Comment:
Split these commands into two lines. Also, can't ajv be run using `ajv-cli`?

Reply:
It can be run either with `ajv` or `ajv-cli`, but regardless it will fail without the last bit due to our use of datetimes in schemas. I will change it `ajv-cli`, but `npx` is a helper with modern `npm` so regardless of choice it will install and execute the libraries default execution handler in one command, not two. :-)

[From earlier this week when I forgot this obvious fact with `-c` and why I need to `npm` in the first place.](https://gitter.im/usnistgov-OSCAL/Lobby?at=62263d32257a3578251a9a21).

Reply:
Also changed locally before pushing back up.